components/raftstore/src/store/peer.rs

```diff
@@ -476,6 +476,140 @@ pub enum ForceLeaderState {
     },
 }
 
+// Following shared states are used while reporting to PD for unsafe recovery and shared among
+// all the regions per their life cycle.
+// The work flow is like:
```

Comment:
please update this

Reply:
Could you please elaborate on what to be updated?

Reply:
plan execution stage

Reply:
PTAL